bundle-size/app.js

```diff
@@ -157,8 +170,19 @@ module.exports = app => {
           output: {
             title: 'Failed to retrieve the bundle size of branch point ' +
                 partialBaseSha,
-            summary: 'bundle size check skipped for this PR. A member of the ' +
-                'bundle-size squad must approve this PR manually.',
+            summary: 'The bundle size (gzipped compressed size of `v0.js`) ' +
+              'of this pull request could not be determined because the base ' +
+              'size (that is, the bundle size of the `master` commit that ' +
+              'this pull request was compared against) was not found in the ' +
+              '`https://github.com/ampproject/amphtml-build-artifacts` ' +
+              'repository. This can happen due to failed or delayed Travis ' +
+              'builds on said `master` commit.\n' +
+              'A member of the bundle-size group will be added automatically ' +
+              'review this PR. Only once this member approves this PR, can ' +
+              'it be merged. If you do not receive a response from this ' +
```

Comment:
nit: Too many uses of the word `this` in this passage :) (Maybe remove one or two?)

Reply:
Done